src/main/java/com/jlr/ttl/ds/api/dto/entity/Location.java

```diff
@@ -0,0 +1,30 @@
+package com.jlr.ttl.ds.api.dto.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "locations")
+public class Location {
```

Comment:
This file is linked to a  DB table and as such should be under the Tables package. You can leave it here for now and move it once I explain how to use the Response, Request  and table interface